[DOCS] CLI Clean-up (#7904)
great-expectations/great_expectations: docs/docusaurus/docs/integrations/integration_datahub.md

#### 3. Run the GX checkpoint

> This Python code can retrieve and run a Checkpoint:

Reviewer: ```suggestion
Run the following command to retrieve and run a Checkpoint:
```
